class/defaults.yml

```diff
@@ -38,6 +38,15 @@ parameters:
       replicaLabels:
         - prometheus_replica
         - rule_replica
+    queryRbacProxy:
+      enabled: false
+      redirectUri: https://${thanos:queryRbacProxy:ingress:host}
```

Comment:
I don't think `redirectUri` needs to be configurable, we can just use `queryRbacProxy.ingress.host` instead.